### DISCUSSION TITLE
bitcoin.conf / lnd.conf user credentials error (unable to create partial chain control: status code: 401, response: "")

### DISCUSSION THREAD

**Discussion by zyklopisch** [lightningnetwork/lnd#7411] [Troubleshooting]
Hello,

i have the following problem - hope that somebody can help me:

I installed lnd, created a wallet, have written down the seed and know i started lnd --> then unlocking the wallet in a separate terminal and I am getting the following error:

![1](https://user-images.githubusercontent.com/101331130/219325315-5a1cacde-8d4c-4937-9906-795fe6dc6492.png)

I think that this has to do with lnd which is not able to login with the rpc-credentials to bitcoind. I am unsure about how to define bitcoind.rpcuser and bitcoind.rpcpass in the lnd.conf, since I have defined the line rpcauth= in bitcoin.conf.

My bitcoin.conf looks like these:

![2](https://user-images.githubusercontent.com/101331130/219326744-5406d758-7d48-4ec0-a505-d601575bc16a.png)

My lnd.conf looks likte these:

![3](https://user-images.githubusercontent.com/101331130/219327596-b98fccee-1891-4c56-954e-5ca68e25e537.png)

Can anyone point me to the solution on how to fix that - what to define in lnd.conf?

lnd and bitcoind runs on the same machine, Ubuntu Server on a notebook, bitcoind is fully synced.

**Comment by guggero** [comment]
How did you create the hash for the `rpcauth` field in `bitcoind`? Are you sure the hashed password matches the one defined in `bitcoind.rpcpass` in `lnd`?

Also, I'm not sure if this could be the issue, but worth a try: You have prefixed config items (for example `routing.strictgraphpruning` within your global section (`[Application Options]`). The config parser doesn't like that and might ignore parts of it.
I'd suggest to remove all (including the top one) section headers, to get a flat config structure.

**Reply by zyklopisch** [reply]
I tried to remove all the things in the config, but this did not solve the problem. I used definitely the same password as bitcoind.rpcpass= written clearly, not hashed like it is in the rpcauth= from bitcoin.conf - that should be correct is my understanding?

**Reply by guggero** [reply]
Yes, in lnd conf you write the clear text password, not the hashed one. Do you have a special character in there that could trip up the config file parser?

**Reply by zyklopisch** [reply]
Thx guggero, that was it! rpcauthy did not recognize the special characters in my password successfully and cut if somehow away.... i created a new one without special characters and now it works!
